src/Solvers/SubSolvers.jl

```diff
@@ -0,0 +1,74 @@
+"""
+    SubSolver
+
+An abstract supertype for structures that contain user-controlled parameters 
+for a linear solver applied to compressed matrix blocks. 
```

Comment:
- Non-first lines should be indented.
- Description should be more generic.
```suggestion
An abstract supertype for structures specifying solutions methods for a linear 
    system or least squares problem.
```

Reply:
corrected